install_with_docker.sh

```diff
@@ -245,7 +245,7 @@ echo -e "ingestPort(8081) and queryPort(80) can be changed using in server.yaml.
 echo -e "queryPort(80) can also be changed by setting the environment variable $PORT."
 
 tput bold
-printf "\n===> ${GREEN_TEXT}Frontend can be accessed on http://localhost:${UI_PORT}${RESET_COLOR}"
+printf "\n===> ${GREEN_TEXT}Frontend can be accessed on http://localhost:${UI_PORT}\a${RESET_COLOR}"
 echo ""
```

Comment:
I see that you have attached the screen shot for this fix. This does not seem to work on MacOs. Are you on a linux machine?

Reply:
@nbhavana  above screenshot in when executed on macOS itself. I will check once again sorry fro inconvenience

Reply:
@nbhavana what's the purpose of this? On mac, in iTerm2 if I hover the link while holding Cmd the link gets highlighted and I can click it to open it in my browser. In Terminal if I hold Cmd and double click the link it also opens it (it doesn't get highlighted, but maybe that's due to some Terminal setting). I haven't tested on linux. This seems to work for me with and without the changes in this PR. Is there a way to make links clickable without needing to hold Cmd?

Reply:
yes you are right, we don't need this change, the link is clickable, if you hold CMD and double click. closing this PR.